{{cookiecutter.app_name}}/README.rst

```diff
@@ -108,3 +108,54 @@ should cache all your assets forever by including the following line
 in your ``settings.py``::
 
     SEND_FILE_MAX_AGE_DEFAULT = 31556926  # one year
+
+Deploying to Heroku::
+
+In order to deploy this to Heroku you need to do a few extra things.  First off, make sure you have already gone through
+the Quickstart above as you will need to run some commands locally to get your migrations initialized before pushing that
+to Heroku.  Also, make sure you have ran git init to initialize your git repo.  Heroku uses git to deploy and when you run
+heroku create below it will add the git remote for you new app.
+
+First, in order to build the static assets using webpack, you need to add a postinstall script to the package.json.  This
+is the preferred approach to building static assets on Heroku.  To do this, simply update the package.json "scripts" section
+with the following::
+
+    "postinstall": "npm run build"
+
+Next, in order to run db migrations as part of the deployment, update the Procfile to look like::
+
+
+    release: flask db upgrade
+    web: gunicorn total_life_challenge.app:create_app\(\) -b 0.0.0.0:$PORT -w 3
+
+You will also need to update your SQLALCHEMY_DATABASE_URI in the Production config to the following::
+
+    SQLALCHEMY_DATABASE_URI = os.environ.get('DATABASE_URL')
```

Comment:
We can update  the ProdConfig so that this manual change is unnecessary.

```python
class ProdConfig(Config):
    SQLALCHEMY_DATABASE_URI = os.environ.get('DATABASE_URL', 'postgresql://localhost/example')

```